src/store/commit.js

```diff
@@ -0,0 +1,35 @@
+import { fetchIndex } from "../api/fetch";
+
+export function index({ commit, auth }, { url, commitAction }) {
```

Comment:
If the `api` `index` is a generator, I don't think this function should still be called `index`. Something like `fetchAll` maybe?